src/operator/nn/fully_connected.cc

```diff
@@ -147,7 +147,8 @@ void FullyConnectedGradComputeExCPU(const nnvm::NodeAttrs& attrs,
                                     const std::vector<NDArray> &inputs,
                                     const std::vector<OpReqType> &req,
                                     const std::vector<NDArray> &outputs) {
-  if (SupportMKLDNNFC(inputs[0])) {
+  bool mkldnn_fc_backward_enable = false;
+  if (SupportMKLDNNFC(inputs[0]) && mkldnn_fc_backward_enable) {
```

Comment:
make a swap:
```
 if (mkldnn_fc_backward_enable && SupportMKLDNNFC(inputs[0]))
```

Reply:
Let's make this change on 1.0 PR directly @rongzha1

Reply:
OK

Reply:
Maybe we can investigate the root cause and try to fix it. Then we don't need this workaround. This can be done following the 1.0 PR merge.